src/System.Runtime.Extensions/src/System/IO/Path.cs

```diff
@@ -188,7 +183,7 @@ public static bool HasExtension(string path)
                     {
                         return i != path.Length - 1;
                     }
-                    if (IsDirectoryOrVolumeSeparator(ch)) break;
+                    if (PathInternal.IsDirectoryOrVolumeSeparator(ch)) break;
```

Comment:
@JeremyKuhne Do you think I should add a `using static PathInternal` at the top here, instead of making all of these changes? (It was necessary since `FindFileNameIndex` used this function, which at the time was a private method in `Path`.)